api/package-lock.json

```diff
@@ -15,11 +15,13 @@
         "bowser": "^2.11.0",
         "buffer-shims": "^1.0.0",
         "compression": "^1.7.4",
+        "cron-validator": "^1.3.1",
```

Comment:
It's a little weird that this is needed here. Since we don't actually alter API code. I'm noticing that there is no update of `shared-libs/transitions/package-lock.json`, but dependencies were added to the package.json file.

Reply:
I noticed there was a recent switch to workspaces. If npm workspaces is like yarn workspaces, there's only supposed to be one lockfile and that is in the root. Here's a link to [npm workspace doc](https://docs.npmjs.com/cli/v8/using-npm/workspaces#:~:text=Below%20is%20a%20post%20npm%20install%20example%2C%20given%20that%20same%20previous%20example%20structure%20of%20files%20and%20folders%3A) insinuating it, but it's not explicitly stated like in the yarn documentation. `api` and `sentinel` are not part of the workspace that's why their lockfiles are being respected.

Reply:
We only use workspaces for shared libs - for actually injecting the shared libs themselves into apps: webapp, api, sentinel. we do

Reply:
I've removed the packages, but CI keeps failing. If you have any ideas on how to fix it, please help me out. Thanks.